source/_posts/rails-api-backend-for-backbone.markdown

```diff
@@ -0,0 +1,337 @@
+
+Since we want to focus on the backend implementation and its integration with the client side application, we decided to borrow the [Backbone TODO application from the TodoMVC project](https://github.com/tastejs/todomvc/tree/gh-pages/examples/backbone).
+
+### Generating the Rails API-only application
+
+Let's get started! First, we need to make sure that we have the most recent version of the Rails source code (remember that, at the time of writing, this feature was just incorporated into the master branch!). The easiest way to have a copy of this code is by cloning the Rails Github project in our computer:
+
+```
+git clone git://github.com/rails/rails.git
+```
+
+Creating a new only-API application is accomplished by adding the `--api` option when generating the project with the `rails new` command. Once Rails 5 is released, you will be able to create the Rails application by running, for example, `rails new my_api_app --api`.
+
+Since we want to generate the application from Rails' source code, the `rails new` command should be a little different, in order to make sure that our generated project will point to our local copy of the Rails source code:
+
+```
+bundle exec railties/exe/rails new <parent-folder-path>/my_api_app --api --edge
+```
+
+It's a good idea to specify a specific path for the generated project, so we avoid to create the Rails API application into the Rails source code folder. That explains the `<parent-folder-path>` placeholder in the example below.
+
+Now, we can explore what is generated in the new project's folder. You will notice that almost everything looks exactly the same than a regular Rails application, and that's certainly true. However, let's highlight what is different.
+
+There are some changes in the generated Gemfile:
+
+```ruby
+source 'https://rubygems.org'
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '5.0.0.alpha'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
+# Use ActiveModel has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
+
+# Use Unicorn as the app server
+# gem 'unicorn'
+
+# Use Capistrano for deployment
+# gem 'capistrano-rails', group: :development
+
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
+
+# Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
+# gem 'rack-cors'
+
+group :development, :test do
+  ....
+  ....
+end
+```
+
+We can see that stuff related with asset pipeline and template rendering is not longer present (TODO: list the gems). In addition, the  `active_model_serializers` is included by default because it will be responsible for serializing the JSON responses returned by our API application.
```

Comment:
That `(TODO: list the gems)` shouldn't be there  or is still pending to add the gem list?